lib/ansible/modules/windows/win_dns_client.ps1

```diff
@@ -52,7 +53,8 @@ Function Get-NetAdapterInfo {
 
     Process {
         if (Get-Command -Name Get-NetAdapter -ErrorAction SilentlyContinue) {
-            $adapter_info = Get-NetAdapter @PSBoundParameters | Select-Object -Property Name, InterfaceIndex
+            $adapter_info = Get-NetAdapter @PSBoundParameters |
+                Select-Object -Property Name, InterfaceIndex, InterfaceGuid
```

Comment:
No need to make this change it just changes the `git blame` when someone wants to track down changes on a line. Also you are retrieving the InterfaceGuid here but the else statement below also needs the GUID

Reply:
Turns out I don't really need the GUID here at all, since it needs to be retrieved again later in either case. Just needed to update a reference to other one at the end to remove this one.